File: articles/algorithmic-risk-management-and-ai-governance/typescript/governance_readiness_score.ts
const scores = [0.60, 0.62, 0.58, 0.52, 0.46, 0.50];
const readiness = scores.reduce((a, b) => a + b, 0) / scores.length;
console.log(`governance_readiness_score=${readiness.toFixed(4)}`);

File: articles/algorithmic-risk-management-and-ai-governance/fortran/governance_readiness_score.f90
program governance_readiness_score
  implicit none
  real, dimension(6) :: scores
  real :: readiness
  scores = (/0.60, 0.62, 0.58, 0.52, 0.46, 0.50/)
  readiness = sum(scores) / 6.0
  print *, "governance_readiness_score=", readiness
end program governance_readiness_score
